code/game/objects/structures/aliens.dm

```diff
@@ -114,36 +250,54 @@
 	density = FALSE
 	layer = ABOVE_OPEN_TURF_LAYER
 	plane = FLOOR_PLANE
-	icon = 'icons/obj/smooth_structures/alien/weeds1.dmi'
-	icon_state = "weeds1"
-	base_icon_state = "weeds1"
+	icon = 'icons/obj/smooth_structures/alien/weeds.dmi'
+	icon_state = "weeds"
+	base_icon_state = "weeds"
 	max_integrity = 15
 	smoothing_flags = SMOOTH_BITMASK
 	smoothing_groups = list(SMOOTH_GROUP_ALIEN_RESIN, SMOOTH_GROUP_ALIEN_WEEDS)
 	canSmoothWith = list(SMOOTH_GROUP_ALIEN_WEEDS, SMOOTH_GROUP_WALLS)
-	transform = matrix(1, 0, -4, 0, 1, -4)
 	var/obj/structure/alien/weeds/node/linked_node = null
 	var/static/list/weedImageCache
-
+	var/check_counter
 
 /obj/structure/alien/weeds/New(pos, node)
 	..()
+	if(src == "Wall Weeeeeed")
+		return
```

Comment:
I'm not sure exactly what you're trying to do here

Reply:
This used to be a check so that wallweeds didn't spred, when wallweeds were a weeds/ subtype
Forgot to remove it, and clean the Wall Weeeeeeeed name for a proper one, done now.